examples/Makefile

```diff
@@ -0,0 +1,49 @@
+.PHONY: default
+default: dev-settings check-credentials
+
+KUSTOMIZE ?= kustomize
```

Comment:
so we're effectively making the standalone kustomize binary a new developer dependency, instead of using the one bundled with kubectl? We should list the dependencies in `examples/README.md`

Reply:
It effectively already was. Note the deliberate use of `?=`, though. You can run make KUSTOMIZE="kubectl kustomize" if you want? I don't think we should be documenting deprecated functionality, though.

Reply:
I've added a section to examples/README.md about required binaries.